src/main/java/ev3dev/utils/DataChannelRereader.java

```diff
@@ -48,17 +48,15 @@ public DataChannelRereader(String pathString) {
     /**
      * @return a string made from the bytes in the file;
      */
-    public String readString() {
+    public synchronized String readString() {
```

Comment:
Rebasing the `channelRewriter` branch on top of `sysfs_perf2` should remove these "virtual" changes. I think these were successfully applied in another PR, but Github still shows them as new changes.

Reply:
If `synchronized` is in sysfs_perf2 then it should /already/ be cleared up, right? I can rebase to see if that clears it.

Reply:
Yes, exactly. I think it has something to do with merge commits - there aren't exactly the same changes in `channelRewriter` as in `sysfs_perf2` commit-wise, so GitHub may not be able to skip changes that were already applied.